torchani/models.py

```diff
@@ -148,8 +148,8 @@ def __getitem__(self, index):
 
         def ase(**kwargs):
             """Attach an ase calculator """
-            from . import ase
```

Comment:
Why the rename?

Reply:
The name causes issues, if you try to import ase from another module it tries to import our ase module first

Reply:
I don't know if this is normal or if there is any other way to avoid it, but for instance doint "imort ase.units" inside nn.py throws an ImportError because torchani.ase.units doesn't exist